projects/1.18/assets/useless-sword/useless_sword/lang/zh_cn.json

```
@@ -0,0 +1,314 @@
+  "item.minecraft.potion.effect.spider_trap_effect_potion_item": "蛛缚药水",
+  "item.minecraft.potion.effect.spidertrapeffect": "蛛缚药水",
+  "item.minecraft.potion.effect.vex_effect": "观察者药水",
+  "item.minecraft.potion.effect.vex_effect_potion_item": "观察者药水",
+  "item.minecraft.potion.effect.vexeffect": "观察者",
+  "item.minecraft.potion.effect.wither_rage": "凋灵之怒",
+  "item.minecraft.potion.effect.wither_rage_potion_item": "凋灵之怒",
+  "item.minecraft.potion.effect.witherrage": "凋灵之怒",
+  "item.minecraft.splash_potion.effect.dizziness": "喷溅型眩晕药水",
+  "item.minecraft.splash_potion.effect.fossilized": "喷溅型石化药水",
+  "item.minecraft.splash_potion.effect.spider_trap_effect": "喷溅型蛛缚药水",
+  "item.minecraft.splash_potion.effect.spider_trap_effect_potion_item": "喷溅型蛛缚药水",
+  "item.minecraft.splash_potion.effect.spidertrapeffect": "喷溅型蛛缚药水",
+  "item.minecraft.splash_potion.effect.vex_effect": "喷溅型观察者药水",
+  "item.minecraft.splash_potion.effect.vex_effect_potion_item": "喷溅型观察者药水",
+  "item.minecraft.splash_potion.effect.vexeffect": "喷溅型观察者药水",
+  "item.minecraft.splash_potion.effect.wither_rage": "喷溅型灵怒药水",
+  "item.minecraft.splash_potion.effect.wither_rage_potion_item": "喷溅型灵怒药水",
+  "item.minecraft.splash_potion.effect.witherrage": "喷溅型灵怒药水",
+  "item.minecraft.tipped_arrow.effect.dizziness": "眩晕之箭",
+  "item.minecraft.tipped_arrow.effect.fossilized": "眩晕之箭",
+  "item.minecraft.tipped_arrow.effect.spider_trap_effect": "蛛缚之箭",
+  "item.minecraft.tipped_arrow.effect.spider_trap_effect_potion_item": "蛛缚之箭",
+  "item.minecraft.tipped_arrow.effect.spidertrapeffect": "蛛缚之箭",
+  "item.minecraft.tipped_arrow.effect.vex_effect": "观察之箭",
+  "item.minecraft.tipped_arrow.effect.vex_effect_potion_item": "观察之箭",
+  "item.minecraft.tipped_arrow.effect.vexeffect": "观察之箭",
+  "item.minecraft.tipped_arrow.effect.wither_rage": "灵怒之箭",
+  "item.minecraft.tipped_arrow.effect.wither_rage_potion_item": "灵怒之箭",
+  "item.minecraft.tipped_arrow.effect.witherrage": "灵怒之箭",
+  "item.nethercatalystsword.name": "§c下界催化剑",
+  "item.netherracksword.name": "下界岩剑",
+  "item.quartzsword.name": "§7石英剑",
+  "item.redsandstonesword.name": "红砂岩剑",
+  "item.sandstonesword.name": "砂岩剑",
+  "item.shulkersword.name": "§d潜影剑",
+  "item.stonecoalsword.name": "石制煤剑",
+  "item.undyinggem.name": "不死宝石",
+  "item.undyingingot.name": "不死锭",
+  "item.undyingsword.name": "§e不死之剑",
+  "item.useless_sword.alloy_sword": "金铁合金剑",
+  "item.useless_sword.amethyst_sword": "紫水晶剑",
+  "item.useless_sword.ancient_guardian_sword": "§e太古守卫者之剑",
+  "item.useless_sword.bamboo_saber": "竹刀",
+  "item.useless_sword.bamboosaber": "竹刀",
+  "item.useless_sword.basalt_broadsword": "玄武岩阔剑",
+  "item.useless_sword.beacon_sword": "§b信标剑",
+  "item.useless_sword.beaconsword": "§b信标剑",
+  "item.useless_sword.big_ghast_tear": "恶魂泪珠",
+  "item.useless_sword.bigghasttear": "恶魂泪珠",
+  "item.useless_sword.brain_coral_sword": "§d脑纹珊瑚剑",
+  "item.useless_sword.braincoralsword": "§d脑纹珊瑚剑",
+  "item.useless_sword.brass_head_turner": "黄铜头槌",
```

```suggestion
  "item.useless_sword.brass_head_turner": "黄铜头槌",
```
游戏内材质貌似是个~~手钻~~，不过应该没问题吧，晚上再来看

---

```suggestion
  "item.useless_sword.brass_head_turner": "黄铜旋转器",
```